installer/common/shared.sh

```diff
@@ -225,15 +225,15 @@ export limitsCpu="2"
 # Installs an NFS server: $1: namespace, $2: boolean onOpenShift
 NFS_SERVER_NAME=cloudflow-nfs
 NFS_CHART_NAME=fdp-nfs
+NFS_CHART_VERSION=0.3.0
 install_nfs_server() {
-helm upgrade $NFS_SERVER_NAME lightbend-helm-charts/$NFS_CHART_NAME \
+helm upgrade $NFS_SERVER_NAME https://github.com/lightbend/${NFS_CHART_NAME}/releases/download/v${NFS_CHART_VERSION}/${NFS_CHART_NAME}-${NFS_CHART_VERSION}.tgz \
```

Comment:
Given that this is a predefined repo URL, I would put the `NFS_CHART_NAME` directly in the URL and not as a separate env var.

Reply:
This is something that pre-existed.